src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -8,20 +8,40 @@
 
 namespace System.Collections.Generic
 {
+    /// <summary>
+    /// A circular doubly linked list.
```

Comment:
@stephentoub let's close this out and wait until the import happens.  Depending on how the process goes, we can also fix things up and incorporate changes as each area has its ///comments imported.